### DISCUSSION TITLE
Nebula-windows-amd64 is detected as Malware by Microsoft defender

### DISCUSSION THREAD

**Discussion by Syncnetworkteam** [slackhq/nebula#899] [Q&A]
Hi Team,

We have planned to implement Nebula in our environment. Microsoft Defender has identified and removed the Nebula latest version [Release v1.7.2] as malware. 

Please see the detection snap for more details.

Is it safe to proceed further? since, even in virus total file is defects malware.

![MicrosoftTeams-image (24)](https://github.com/slackhq/nebula/assets/136309193/f6bac9bd-e910-4ea1-8709-a1ae559d7738)
![MicrosoftTeams-image (23)](https://github.com/slackhq/nebula/assets/136309193/7ac9efd9-165b-460c-a303-e48c694ccb7f)
![MicrosoftTeams-image (25)](https://github.com/slackhq/nebula/assets/136309193/e6ef2dbb-13ec-42af-a4f1-d359e871c295)

**Comment by johnmaguire** [comment]
Closing this as a dupe of #898.
